### PR TITLE
Add appmap.labels decorator

### LINKED ISSUE #61: django.dispatch crashes the appmap client.
When recording chipy_org with the Django package included in appmap.yml, the client crashes a test unless django.dispatch is excluded.

### PR DESCRIPTION
**Pull request by apotterri**
Provide the `labels` decorator to allow the user to add annotations
(labels) to the classMap entries for functions.

Also, get rid of some code that modified an instrumented function's kwargs. Fixes #61.

### CODE REVIEW

**Review by dividedmind — APPROVED** [review]
Looks good, thanks! If it's not too much work, please spin off the README formatting changes to a separate commit before merging. And the kwargs fix needs a changelog entry :)

File: appmap/_implementation/event.py [hunk 1]
```diff
@@ -45,6 +45,15 @@ def reset(cls):
...
 
+def describe_value(val):
+    """
+    Return a description of the given value, with the form
+    (class_name, object_id).
+    """
+    return (fqname(type(val)),
+            id(val))
+
+
```
Comment (dividedmind) [inline appmap/_implementation/event.py:56]: How about instead returning a `{class, value, object_id}` dict and then use that to DRY up `Param` and `FuncReturnEvent`?

**Review by dividedmind — CHANGES_REQUESTED** [review]
Good catch! I have a suggestion for a slight improvement.

File: appmap/_implementation/event.py (hunk 1 above)
Comment (apotterri) [reply appmap/_implementation/event.py:56]: Will do.

**Review by dividedmind — APPROVED** [review]
Thanks!

**Pull request merged**